### PR TITLE
fix(DB/Creature): Plundering the Plunderers - Polly improvements.

### PR DESCRIPTION
**Pull request by avarishd**
## Changes Proposed:
- Make the RP blizzlike

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/13132
- Closes https://github.com/chromiecraft/chromiecraft/issues/4067

## SOURCE:

https://youtu.be/LzmXDQ1DlyE?t=53
https://youtu.be/gZyx81kIXG4?t=220
## Tests Performed:

- tested

## How to Test the Changes:

make a rogue
.level 19
.q a 2381
.go xyz -1456 -3959 0 1
.learn 1804
.maxskil
.additem 7970
Click the box, confirm that the summoned Polly is level 50
Use the the item
Compare the provided source to AC.

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

**Comment by Maelthyrr** [comment]
Hi, thanks for the PR. Look fine:

- [X] Creature spawn with the level 50
- [X] Item transform the creature to 18 lvl

Just the spawn position seems not correct based on your sources but it not related to your PR and needs a sniff to get the correct position. I will create an issue for this.

**Comment by avarishd** [comment]
> Just the spawn position seems not correct based on your sources but it not related to your PR and needs a sniff to get the correct position. I will create an issue for this.

Oh yeah, also missing that yell. Will add that from this PR.

**Comment by avarishd** [comment]
@Maelthyrr I'm sorry, but can you test again please?

It attacks in stealth (the invoker), as from the source. 
Corrected the spawn location to be like the source (checked few videos, it's between the 1st and 2nd barrel) 
If it (the lvl 50 one) doesn't die OR doesn't enter combat (player DC/.gm on) it will go to the previous location and stealth there.

<details>

https://user-images.githubusercontent.com/46330494/223504085-cf503e8f-08d6-4c3e-a967-43b2f604f33d.mp4

https://user-images.githubusercontent.com/46330494/223512433-88bf0e1b-d1f5-40c0-9e62-894d484abfb6.mp4

https://user-images.githubusercontent.com/46330494/223515033-f261f266-6058-4e3c-a6de-388f53f2f1bc.mp4


</details>

**Comment by Gultask** [comment]
I was wondering before but the only other core to use level 18 is cmangos' Classic branch. Level 50 is the correct one.

**Comment by Maelthyrr** [comment]
Every

> @Maelthyrr I'm sorry, but can you test again please?
> 
> It attacks in stealth (the invoker), as from the source. Corrected the spawn location to be like the source (checked few videos, it's between the 1st and 2nd barrel) If it (the lvl 50 one) doesn't die (doesn't get hit with E.C.A.C) OR doesn't enter combat (player DC/.gm on) it will go to the previous location and stealth there.

Everything is fine now, and look as your sources.

File: data/sql/updates/pending_db_world/rev_1677938661497953700.sql [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+(7167, 0, 0, 1, 54, 0, 100, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Polly - On Just Summoned - Say Line 0'),
+(7167, 0, 1, 2, 61, 0, 100, 0, 0, 0, 0, 0, 0, 49, 0, 0, 0, 0, 0, 0, 7, 0, 0, 0, 0, 0, 0, 0, 0, 'Polly - On Just Summoned - Start Attacking'),
+(7167, 0, 2, 0, 61, 0, 100, 0, 0, 0, 0, 0, 0, 101, 0, 0, 0, 0, 0, 0, 8, 0, 0, 0, 0, -1456.55, -3959.53, 0.24, 1.99, 'Polly - On Just Summoned - Set Home Position'),
+(7167, 0, 3, 0, 1, 0, 100, 1, 73000, 73000, 60000, 60000, 0, 11, 8822, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Polly - OOC - Cast Stealth'),
+(7167, 0, 4, 0, 1, 0, 100, 257, 5000, 5000, 0, 0, 0, 69, 0, 0, 0, 0, 0, 0, 8, 0, 0, 0, 0, -1456.55, -3959.53, 0.24, 1.99, 'Polly - Out of Combat - Move To Position (No Repeat/Reset)'),
```
Comment (Gultask) [inline data/sql/updates/pending_db_world/rev_1677938661497953700.sql:15]: What's the source for these positions?

Comment (avarishd) [reply data/sql/updates/pending_db_world/rev_1677938661497953700.sql:15]: The move OOC is the pre-pr spawn position (next to chest), the event_scripts one is from the  2 video sources that are linked.

**Review by Gultask — APPROVED** [review]

**Pull request merged**